classes/ActionScheduler_AdminView.php

```diff
@@ -35,7 +50,7 @@ public static function instance() {
 	 * @codeCoverageIgnore
 	 */
 	public function init() {
-		if ( is_admin() && ( ! defined( 'DOING_AJAX' ) || false == DOING_AJAX ) ) {
+		if ( is_admin() && ( ! defined( 'DOING_AJAX' ) || false === DOING_AJAX ) ) {
```

Comment:
```suggestion
		if ( is_admin() && ( ! defined( 'DOING_AJAX' ) || ! DOING_AJAX ) ) {
```

I think this might be the safer way to solve for `WordPress.PHP.StrictComparisons.LooseComparison` in this particular scenario. For instance, it doesn't seem impossible that someone somewhere will have code like this:

```php
define( 'DOING_AJAX', 0 );
```